Task3/src/main/java/DispatchContainersValidator.java

```diff
@@ -0,0 +1,16 @@
+public class DispatchContainersValidator implements Validator {
+
+    @Override
+    public boolean validate(ValidationInformation validationInformation) {
+        boolean areDispatchContainersEmpty = validationInformation
+                .getFloors()
+                .stream()
+                .allMatch(Floor::isDispatchContainerEmpty);
+        if (areDispatchContainersEmpty) {
+            Validators.LOGGER.info("All dispatchContainers are empty - \u2713");
```

Comment:
What's the difference we have between if and else statements? Could we move logic regarding logs outside these statements? I mean we can do something like this:
```
String value;
if (condition) {
value = ...
} else {
value = ...
}
Validators.LOGGER.info("All dispatchContainers are empty - " + value);
```